docs/reference/commandline/stack_ps.md

```diff
@@ -36,8 +36,30 @@ command has to be run targeting a manager node.
 
 ## Examples
 
+### List the tasks that are part of a stack
+
+The following command shows all the tasks that are part of the `voting` stack:
+
 ```bash
-$ docker stack ps
+$ docker stack ps voting
+ID                  NAME                  IMAGE                                          NODE
+ DESIRED STATE       CURRENT STATE           ERROR               PORTS
+xim5bcqtgk1b        voting_worker.1       dockersamples/examplevotingapp_worker:latest   node2
```

Comment:
It looks like you wrapped the output in your examples. Although we _do_ wrap output in other parts, the examples should not be wrapped, but included as-is